.circleci/config.yml

```diff
@@ -24,6 +71,10 @@ jobs:
             echo ${DOCKERHUB_SECRET} | docker login -u ${DOCKERHUB_USER} --password-stdin
             sudo apt update
             sudo apt install make bash git
+      - run:
+          name: Copy binaries from workspace
+          command: |
+            cp -r /build-* .
```

Comment:
We will have:
* build-amd64/driverkit
* build-arm64/driverkit

The Dockerfile adds the correct one based on arg TARGETARCH (automatically provided by docker buildx).